src/random.jl

```diff
@@ -114,3 +114,69 @@ Return BitVector of primary producers.
 function whoisproducer(A)
     vec(.!any(A, dims=2))
 end
+
+"""
+**Create facilitation links**
+
+Takes trophic matrix (A) and number or connectance of facilitation links (resp. L and C).
+Returns matrix of facilitation links.
+Facilitating species are in rows, facilitated species are in columns.
+"""
+function facilitationlinks(A, L::Int64)
+
+    # Test.
+    @assert L >= 0
+
+    # Find candidate links.
+    S = numberspecies(A)
+    facilitation_matrix = zeros(Int, S, S) # initialize
+    candidates = wherecanfacilitate(A)
+    size(candidates, 1) < L ? throw(ArgumentError("L is too large.")) : nothing
```

Comment:
Julia idiom in this case: `size(..) < L && throw(...)`. Read [here](https://docs.julialang.org/en/v1/manual/control-flow/#Short-Circuit-Evaluation) ;)